app/addons/documents/changes/components.react.jsx

```diff
@@ -270,7 +270,7 @@ class ChangeRow extends React.Component {
   render () {
     const { codeVisible } = this.state;
     const { change, databaseName } = this.props;
-    const jsonBtnClasses = 'btn btn-small' + (codeVisible ? ' btn-secondary' : ' btn-info');
+    const jsonBtnClasses = 'btn btn-small' + (codeVisible ? ' btn-secondary' : ' btn-secondary');
```

Comment:
We can simplify this now:

`const jsonBtnClasses = 'btn btn-small btn-seconday'`